Assets/Scripts/Data/Main/PlayerData.cs

```diff
@@ -3,8 +3,11 @@
  * Description: Stores data about the player
  */
 
+using System.Collections;
+using UnityEngine;
+
 [System.Serializable]
-public class PlayerData : IPlayerData {
+public class PlayerData : MonoBehaviour, IPlayerData {
```

Comment:
We don't want PlayerData to be a MonoBehaviour. This means that it can't be initialized with a constructor and that it has to be associated with a GameObject (which messes up the structure of the DataController system.)